frontend/src/stores/applications.js

```diff
@@ -74,5 +78,16 @@ export const useApplicationsStore = defineStore('applications', {
         throw error
       }
     },
+
+    async getSupplementaryApplications(applicationId) {
```

Comment:
If we don't need to save any data to store, I think we can remove this action and just use ApplicationService.getSupplementaryApplications(applicationId)